### PR TITLE
Adding ivan to developer list.

### PR DESCRIPTION
**Pull request by evert**
@Hywan are you ok with this change?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,6 +25,12 @@
             "email" : "dominik@fruux.com",
             "homepage" : "http://tobschall.de/",
             "role" : "Developer"
+        },
+        {
+            "name" : "Ivan Enderlin",
+            "email" : "ivan.enderlin@hoa-project.net",
+            "homepage" : "http://cv.mnt.io/",
```
Comment (Hywan) [inline composer.json]: Just http://mnt.io/ ;-).

**Comment by Hywan** [comment]
Thanks!

**Comment by Hywan** [comment]
Just, I am fine with my `ivan.enderlin@fruux.com` address email too.

**Comment by evert** [comment]
Your personal email address is more likely your canonical identity :) Will change the url though

**Pull request merged**